packages/sampler/stories/default/ProgressBar.js

```diff
@@ -15,20 +15,25 @@ storiesOf('Moonstone', module)
 	.add(
 		'ProgressBar',
 		() => {
-			const side = select('side', ['after', 'before', 'left', 'right'], ProgressBarTooltipConfig, 'before');
+			// added here to force Storybook to put the ProgressBar tab first
+			const disabled = boolean('disabled', ProgressBarConfig);
```

Comment:
This is a bummer we need to do this but I guess it makes sense.  We can fix other stories (if any) later that have a similar problem